### PR TITLE
add missed onnx ops & bug fix

### PR DESCRIPTION
**Pull request by mzmssg**

### CODE REVIEW

File: src/nnfusion/core/operators/generic_op/generic_op_define/Sum.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@ REGISTER_OP(Sum)
         std::string dtype;
         NNFUSION_CHECK(
             element::Type::nnfusion_element_type_to_dtype_string(curr->get_element_type(), dtype));
-        NNFUSION_CHECK(dtype == "float32");
+        // NNFUSION_CHECK(dtype == "float32") << "unsupported Sum dtype " << dtype;
```
Comment (mzmssg) [inline src/nnfusion/core/operators/generic_op/generic_op_define/Sum.cpp:17]: need confirmed `int64` type support

File: src/nnfusion/core/kernels/cuda_gpu/cuda_cudnn.cpp [hunk 2]
```diff
@@ -178,13 +178,15 @@ LanguageUnit_p cuda::get_cudnn_convolution_descriptor(const Shape& padding,
...
                 ss << to_string(d[i]) << ", ";
-            ss << to_string(d.back()) << "}\n";
+            ss << to_string(d.back()) << "}";
             return ss.str();
         };
 
-        expand_vector_int("padding_int", padding_int);
-        expand_vector_int("window_movement_strides_int", window_movement_strides_int);
-        expand_vector_int("window_dilation_strides_int", window_dilation_strides_int);
+        lu << expand_vector_int("padding_int", padding_int) << ";\n";
```
Comment (mzmssg) [inline src/nnfusion/core/kernels/cuda_gpu/cuda_cudnn.cpp:185]: still report error
image

**Comment by xysmlx** [comment]
Should other required ONNX OPs be implemented in this PR?

**Comment by mzmssg** [comment]
> Should other required ONNX OPs be implemented in this PR?

Nope, the others should be after folder refactor

**Review by xysmlx — APPROVED** [review]

**Pull request merged**